bin/workbench-daemon.sh

```diff
@@ -148,15 +148,15 @@ function find_workbench_process() {
   pid=`found_workbench_server_pid`
 
   if [[ -z "$pid" ]]; then
-    if ! kill -0 ${pid} > /dev/null 2>&1; then
+    echo "${WORKBENCH_NAME} is not running"
+    return 1
+  else
+   if ! kill -0 ${pid} > /dev/null 2>&1; then
```

Comment:
The first letter needs to be indented into two.

Reply:
> The first letter needs to be indented into two.

Done, thanks to point it